cmd/dranet/app.go

```diff
@@ -47,18 +47,18 @@ const (
 )
 
 var (
-	hostnameOverride string
-	kubeconfig       string
-	bindAddress      string
-	celExpression    string
+	nodeNameFlag  string
+	kubeconfig    string
+	bindAddress   string
+	celExpression string
 
 	ready atomic.Bool
 )
 
 func init() {
-	flag.StringVar(&kubeconfig, "kubeconfig", "", "absolute path to the kubeconfig file")
+	flag.StringVar(&kubeconfig, "kubeconfig", "", "Absolute path to the kubeconfig file. If not specified, the in-cluster config will be used.")
 	flag.StringVar(&bindAddress, "bind-address", ":9177", "The IP address and port for the metrics and healthz server to serve on")
-	flag.StringVar(&hostnameOverride, "hostname-override", "", "If non-empty, will be used as the name of the Node that kube-network-policies is running on. If unset, the node name is assumed to be the same as the node's hostname.")
```

Comment:
this is how works in kubelet, kube-proxy and most of the components that depend on this ... I do not think we need to deviate and keep it this way, it is a mess and confusing https://github.com/kubernetes/kubernetes/pull/126896  but it is what it is ... we can just replace the manifest